src/logger.c

```diff
@@ -125,6 +128,8 @@ void logger(int level, int priority, const char *format, ...) {
 	va_list ap;
 	char message[1024] = "";
 
+	if(!should_log(level)) return;
```

Comment:
Please run `make astyle` from the top directory.

Reply:
done